pkg/apis/cluster/types.go

```diff
@@ -104,6 +125,97 @@ type ClusterSpec struct {
 	// any resource that does not tolerate the Taint.
 	// +optional
 	Taints []corev1.Taint
+
+	// ResourceModels is the list of resource modeling in this cluster. Each modeling quota can be customized by the user.
+	// Modeling name must be one of the following: cpu, memory, storage, ephemeral-storage.
+	// If the user does not define the modeling name and modeling quota, it will be the default model.
+	// The default model index from 0 to 8. Each default model's cpu quota is [2^(index-1), 2^index), 2 <= index <= 8
```

Comment:
when index =0， 2^(0-1)?， and memory quota doesn't satisfy your formula.

Reply:
When index = 0 or index = 1, it does not have a uniform formula. So the formula starts with index = 2.